[DEVX-161] Improve finance api docs (#118)
iZettle/api-documentation: finance-api/overview.md

Finance API overview
===

> Merchants taking payments with Zettle have internal accounts that handle transactions for the payments. For example, card payments made with Zettle card readers.

Reviewer: I found this part bit hard to understand. This feels a bit confusing to me. 
In my opinion, I think it's best to explain things simple as possible in simple English.
Not sure what is the best way to phrase this, maybe something like this

"The Finance API is used to fetch finance related information processed through Zettle this includes account balance, all the transactions process through Zettle and payout information"

What do you think?

> For merchants who take payments with Zettle, they have a Zettle account that handles transactions for the payments. For example, card payments made with a Zettle card reader.

Reviewer 2: Who are "they"?

```suggestion
Merchants taking payments with Zettle have internal accounts that handle transactions for the payments. For example, card payments made with Zettle card readers.
```

> The Finance API fetches transaction information about the payments.

Reviewer 2: ```suggestion
The Finance API is used to fetch information about the transactions.
```

> # Understand how card payments work at Zettle

Reviewer 3: Not very important, but what about dropping the `card` here and just saying how a `payment` works?
The flow describes card payments, card refunds and APMs. All of them work the same way. Maybe integrators wouldn't be in doubt about APMs and refunds work.

Finance API overview
===
The Finance API fetches information about transactions that are made through Zettle. For example, card payments made with a Zettle card reader.

> # Understand how card payments work at Zettle

Reviewer 4: I don't think that "how card payments work at Zettle" is relevant in the finance context; for the purposes of the finance API, it seems to me that people need to understand what the accounts mean (preliminary, liquid) and perhaps where the money on the preliminary/liquid account could come from--card present payments, payment links, QRC payments... depending on the market, of course. 

So I would propose removing "How card payments work" from the Finance API docs entirely. 

What do you think @Astro-Choco?

> For every merchant who makes transactions with Zettle, they have an Zettle account that handles those transactions.

Reviewer 2: This sentence is hard to understand. Who are they for example?

I think we should drop this block since it reveals our internal architecture.

Finance API overview
===
Merchants who take payments with Zettle have an Zettle account that processes transactions for the payments. For example, card payments made with Zettle card readers.

The Finance API is used to fetch finance-related information that is processed through Zettle. The information includes account balance, all the transactions processed through Zettle, and payout.

# Understand how payments work at Zettle

> Payments are card payments, card refunds, or alternative payment methods (APMs) like PayPal Wallet.

Reviewer 3: PayPal Wallet is not a payment, it's like a bank account. PayPal QRC is.
